### PR TITLE
fix #91: created a login page for SAML testing

### LINKED ISSUE #91: Create a dummy HTML site for SAML2 Testing
Need a dummy SAML2 site for SAML2 testing this can be done with simple HTML and javascript and is used for testing purposes only.

### PR DESCRIPTION
**Pull request by agarcia-caicedo**
Created a simple html login page for SAML testing purposes

### CODE REVIEW

**Comment by Grommers00** [comment]
Looks good - very simple clean code,  hopefully  we can eventually add in more to use this as the base for our tests regarding saml2.  This may need to be updated later it looks like to reflect seneca's redirect page.

File: src/login/login.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<form>
+  
+    <div class="container">
```
Comment (neilong31) [inline src/login/login.html]: Maybe we can add a h1 tag that tells the user to login or something?

File: src/login/login.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<form>
+  
+    <div class="container">
+      <label for="uname"><b>Email</b></label>
+      <input type="text" placeholder="Enter Email" name="ename" required>
```
Comment (neilong31) [inline src/login/login.html]: Could we also add some comments explaining what things do for future students?

**Comment by neilong31** [comment]
It looks good tho and shouldn't break anything :)

File: src/login/login.html (hunk 2 above)
Comment (agarcia-caicedo) [reply src/login/login.html]: I added some comments, thanks for the feedback

File: src/login/login.html (hunk 1 above)
Comment (agarcia-caicedo) [reply src/login/login.html]: As of right now, the login page is for testing purposes only, so I feel the tag would superfluous for this

**Pull request merged**